README.md

```diff
@@ -189,8 +199,8 @@ Rancher can be built from source on Windows, macOS or Linux.
 Cross-compilation is currently not supported. To run a build do:
 
 ```
-npm run build
-npm run package --publish=never
+yarn build
+yarn package --publish=never
```

Comment:
Further down, at line 222, there's a reference to `MacOS`. It should be `macOS`

Reply:
And line 308, `of how its` should be `of how it's`

Reply:
I can follow up to fix that. I think it's unrelated to this PR.